CHANGELOG.md

```diff
@@ -1,4 +1,67 @@
 # osquery Changelog
+<a name="5.5.0"></a>
+## [5.5.0](https://github.com/osquery/osquery/releases/tag/5.5.0)
```

Comment:
I don't know how you handled changelog for point-releases in the past but for 5.5.1:

`libs: Update sqlite to version 3.39.2 (https://github.com/osquery/osquery/pull/7736)`

Reply:
And

- `Add cgroup_path column to processes table on Linux (#7728)`
- `Update platform_info schema to handle platform specific columns`
- `Move process_open_sockets.state to normal schema`

Reply:
Updated this for 5.5.1. 

Since 5.5.0 never went stable, I'm going to pretend like it never existed